doc/security/CVE-2022-0670.rst

```diff
@@ -0,0 +1,43 @@
+.. _CVE-2022-0670:
+
+CVE-2022-0670: Native-CephFS Manila Path-restriction bypass
+===========================================================
+
+Summary
+-------
+
+Users who were running OpenStack Manila to export native CephFS, who
```

Comment:
@zdover23 I would remove the comma(s) and add an _and_ before _who upgraded_.

Reply:
These changes have been made.